src/VNC_ILI9341.h

```diff
@@ -27,10 +27,10 @@
 #ifndef VNC_ILI9341_H_
 #define VNC_ILI9341_H_
 
-#include "VNC_config.h"
-
 #ifdef VNC_ILI9341
 
+#include "VNC_config.h"

```

Comment:
looks like that this change is breaking all examples which use the ILI9341.

Reply:
🤦‍♀️
I not a fan of config file within libraries and with platformio it wasn't an issue. I reverted the change